src/plugins/syscalls/linux.h

```diff
@@ -1601,8 +1613,346 @@ static const syscall_t* linux_syscalls_table[] =
     [332] = &statx,
 };
 
+static const syscall_t* linux_syscalls_table_x64[] =
```

Comment:
Perhaps a comment explaining the differences between this and the `x32` variant would be nice so its easier to see why we have two separate tables.